### ISSUE TITLE
Labeling Ground Truth

### ISSUE THREAD

**Issue by nuh-sikander** [mpicbg-csbd/stardist#27]
Hi,
As it says;

> The training data that needs to be provided for StarDist consists of corresponding pairs of raw images and pixelwise annotated ground truth images (masks), where every pixel has a unique integer value indicating the object id (or 0 for background). 

Can you please shed some light on how to do that for your own data-set? I can see a lot of annotation/masking tools but having hard time getting the required output.

Also, you said this is a subset of kaggle DSB2018. How you guys converted that format to this one?

Help is much appreciated.

**Comment by maweigert** [comment]
> Can you please shed some light on how to do that for your own data-set? I can see a lot of annotation/masking tools but having hard time getting the required output.

We typically use the [Labkit plugin](https://imagej.net/Labkit) from within [Fiji](https://fiji.sc/) to create the label images (in 2D or 3D).  

> Also, you said this is a subset of kaggle DSB2018. How you guys converted that format to this one?

Yes, we converted the [original data](https://fiji.sc/) (where a png image was provided per indivual label) to a single integer valued stack.

Hope that helps!

**Comment by nuh-sikander** [comment]
Thanks a lot @maweigert. Much appriciated.
